packages/test/test-end-to-end-tests/src/test/gc/gcSweepDataStores.spec.ts

```diff
@@ -173,18 +169,36 @@ describeCompat("GC data store sweep tests", "NoCompat", (getTestObjectProvider)
 		};
 	};
 
-	const setupContainerCloseErrorValidation = (container: IContainer) => {
-		container.on("closed", (error) => {
-			assert(error !== undefined, `Expecting an error!`);
-			assert(error.message.startsWith("DataStore was deleted:"));
-		});
+	/**
+	 * Returns a promise that will resolve when all the passed in containers have closed dur
+	 * to "DataStore was deleted" error.
+	 */
+	const getContainersClosePromise = async (containers: IContainer[]) => {
+		const closePromises: Promise<void>[] = [];
+		for (const container of containers) {
+			const promise = timeoutPromise((resolve) => {
```

Comment:
You're not passing any timeout options, why use timeoutPromise? What does it to in this case?

Reply:
I had similar doubt some time back and updated the comment here to explain it - https://github.com/microsoft/FluidFramework/blob/7ae839dc1388320f4c34e5aa941ae0e67d5cefde/packages/test/test-utils/src/timeoutUtils.ts#L157.